app/static/js/tickets/prepay-tickets.js

```diff
@@ -0,0 +1,19 @@
+$('.autofill-checkbox').click(function(){
+        var firstName = $('#firstname').val();
+        var lastname =  $('#lastname').val();
+        var email =     $('#email').val();
+        var requiredId = $(this).data('ticket-type') + '-' + this.id ;
+        if($(this).is(':checked')) {
+            $(document.getElementById(requiredId + '-holders[firstname]')).val(firstName);
```

Comment:
@niranjan94 document.getElementById was deliberately used instead of jquery selector because of the square brackets in the id of the input fields. Since they have a special meaning to jquery selectors, it can't be used. I tried escaping them with \\ but it didn't work either. Hence I resorted to DOM selector. It's a bad practice to use square braces in the ID of the elements if we are using jquery, but I didn't change the id as it might have had implications in other files as well. What course do you recommend , to continue with it or change the ID?